Makefile

```diff
@@ -6,10 +6,17 @@ build:
 	export GOBIN=${ROOT_PATH}/bin && \
 	go install ./...
 
-test: build
+test:
 	export GOBIN=${ROOT_PATH}/bin && \
 	go test --race -timeout 2m ./...
 
 clean:
 	go clean -i ./...
 	@rm -rf ./bin
+
+update_vendor:
+	which glide >/dev/null || curl https://glide.sh/get | sh
+	which glide-vc || go get -v -u github.com/sgotti/glide-vc
+	glide --verbose update --strip-vendor 
```

Comment:
I tried this command, but got an error `--strip-vendor cannot be used without --strip-vcs`
@overvenus any suggestion?